index.html

```diff
@@ -43,6 +43,153 @@ <h3>LET'S CONNECT</h3>
+                    <p>Canopy.Back-end.2023</p>
+                </div>
+                <p class="project_description">A daily selection of privately personalized reads; no 
+                    accounts or sign-ups required.</p>
+                <footer>
+                    <ul class="skills">
+                        <li>Html</li>
+                        <li>Css</li>
+                        <li>Javascript</li>
+                    </ul>
+                    <div class="external_link">
+                        <a href="#">see project</a>
+                    </div>
+                </footer>
+            </div>
+        </div>
+        <div class="project">
+            <img class="project_image" src="assets/Snapshoot Portfolio (3).png" alt="">
+            <div class="general">
+                <h3>Multi-post Stories</h3>
+                <div class="general_inf">
+                    <p>Canopy.Back-end.2023</p>
+                </div>
+                <p class="project_description">A daily selection of privately personalized reads; no 
+                    accounts or sign-ups required.</p>
+                <footer>
+                    <ul class="skills">
+                        <li>Html</li>
+                        <li>Css</li>
+                        <li>Javascript</li>
+                    </ul>
+                    <div class="external_link">
+                        <a href="#">see project</a>
+                    </div>
+                </footer>
+            </div>
+        </div>
+        
+
+    </div>
+    <section class="last-sect">
+        <section id="about-myself">
+        <div class="myself-abt">
+            <h1>About<br>Myself</h1>
+            <p id="presentation">I’m a software developer! I can help you build a product , 
+                feature or website Look through some of my work and experience! 
+                If you like what you see and have a project you need coded, 
+                on’t hestiate to contact me.
+            </p>
+            <div id="resume-link">
+                <h2>LETS CONNECT</h2>
+                <ul>
+                    <li><a href="https://www.linkedin.com/in/venkatesh-shri-sakthi/" ><img src="assets/Linkedin icon.png" ></a></li>
+                    <li><a href="https://github.com/shri-sakthi-v"><img src="assets/Vector (1).png" ></a></li>
+                    <li><a href="https://twitter.com/shrisakthi"><img src="assets/Vector.png" ></a></li>
+                    <li><a href="https://twitter.com/shrisakthi"><img src="assets/storm.png" alt="#"></a></li>
+                    <li><a href="https://twitter.com/shrisakthi"><img src="assets/mail.png" alt=""></a></li>
+                </ul>
+                <a href="#" id="get-resume">Get my resume</a>
+            </div>
+        </div>
+        <div id="my_skills">
+            <div id="languages_skills">
+                <div class="spec_name">
+                    <p>Languages</p>
+                    <div class="show-language">
```

Comment:
Kindly update your languages section to include the three technologies in the screenshot below(HTML, CSS, JS)🙏

<img width="209" alt="image" src="https://user-images.githubusercontent.com/37116322/219726606-5a5fc7a1-6455-4d0e-967f-3dcc6ab94bb0.png">

- Also, please update each section to have the correct text from the figma template
- Please also try to align the text and the arrow button so that they can be on the same line

Reply:
added the javascript and html and css  technology .